src/main/java/pro/sky/cookingappnew/controllers/FileController.java

```diff
@@ -0,0 +1,111 @@
+package pro.sky.cookingappnew.controllers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.v3.oas.annotations.Operation;
+import org.apache.commons.io.IOUtils;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.*;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import pro.sky.cookingappnew.model.Ingridients;
+import pro.sky.cookingappnew.model.Recipe;
+import pro.sky.cookingappnew.services.ExportRecipeService;
+import pro.sky.cookingappnew.services.FileService;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/files")
+public class FileController {
+
+    private FileService fileService;
+    private ObjectMapper objectMapper;
+    private FileController exportRecipeService;
+    private final static String STORE_FILE_NAME = "recipes";
+    private Map<Integer, Recipe> recipes = new HashMap<>();
+
+    public FileController(FileService fileService) {
+        this.fileService = fileService;
+    }
```

Comment:
Соответственно, нам нужно заинжектить recipeService через конструктор

```suggestion
    public FileController(FileService fileService,RecipeService recipeService) {
        this.fileService = fileService;
        this.recipeService = recipeService;
    }
```